rotkehlchen/types.py

```diff
@@ -212,7 +213,7 @@ class ChainID(Enum):
     OPTIMISM = 10
     BINANCE = 56
     GNOSIS = 100
-    MATIC = 137
+    POLYGON_POS = 137
```

Comment:
Do we use the string version? I see that we do it when serializing assets so it might be an issue to change this without caution. Have you checked the places where this might create an issue?

Reply:
From what I checked it has no issues since the only usage of this constant was in assets mappings